packages/instrumentation-mongodb/package.json

```diff
@@ -16,12 +16,13 @@
     "compile:with-dependencies": "nx run-many -t compile -p @opentelemetry/instrumentation-mongodb",
     "compile": "tsc -p .",
     "prepublishOnly": "npm run compile",
-    "tdd": "npm run test-v5-v6-run -- --watch-extensions ts --watch",
```

Comment:
I dropped the 'tdd' script. It has been broken since #2932 (5 months ago). It isn't a script that we typically have/promise-to-have in all the packages.